client/landing/gutenboarding/components/header/style.scss

```diff
@@ -81,6 +84,5 @@ $padding--gutenboarding__header-domain-picker-button: 0.3em 0.5em;
 }
 
 .gutenboarding__site-title {
-	padding: $padding--gutenboarding__header-domain-picker-button;
```

Comment:
With this removal, the variable is used in 1 place and we can use the value directly. Do you mind adding that change?